orttraining/orttraining/python/training/onnxblock/engine/training_module.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+from onnxruntime.capi import _pybind_state as C
+from onnxruntime.capi.onnxruntime_pybind11_state import OrtValueVector
+
+
+class TrainingModule:
+    """
+    Class for running Training.
+    This class is a wrapper of Module Class.
+    """
+
+    training: bool
+
+    def __init__(
+        self, train_model_uri, ckpt_uri, eval_model_uri=None, session_options=None, providers=None, **kwargs
```

Comment:
Let's provide suitable defaults for session options and providers and not expose the in the constructor. We also don't need kwargs right now.